modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -32,12 +32,12 @@ In the example above, _Bob_ is the administrator responsible for deciding which
 Meanwhile, _Alice_ is the administrator that assigns who has access to the privileges set by _Bob_.
 In the example, _Alice_ will assign that access to _Carol_.
 
-See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/dbms-administration[Cypher manual -> DBMS administration] for more information.
+See lxref:authentication-authorization/dbms-administration.adoc[Cypher manual -> DBMS administration] for more information.
```

Comment:
if the link no longer points to Cypher Manual, should the link text also be updated?
```suggestion
See lxref:authentication-authorization/dbms-administration.adoc[DBMS administration] for more information.
```